### PR TITLE
fix(app): add USB order to protocol module list

### PR DESCRIPTION
**Pull request by ahiuchingau**
# Overview
Adding the USB order to the module list in order to help user figure out which port to use when
plugging in modules of the same type.

# Changelog
- update module deck map copy
- add USB port order in calibrate panel (and only show this when multiple modules of the same type are required)
- remove module list check icon

### CODE REVIEW

**Review by nusrat813 — APPROVED** [review]
LGTM

File: app/src/assets/localization/en/protocol_calibration.json [hunk 1]
```diff
@@ -7,12 +7,14 @@
   "labware_cal_title": "Labware Calibration",
   "labware_cal_tipracks_title": "tipracks",
   "labware_cal_labware_title": "labware",
-  "module_connect_description": "Plug in and power up the required module(s) via the OT-2 USB Ports. Place the modules as show in the deck map.",
-  "module_connect_duplicate_description": "Duplicate modules should be plugged in to the USB ports with the lowest deck slot number corresponding to the left-most USB port. <a>Check out our help docs</a> for more information on using duplicate modules.",
+  "module_connect_instruction": "Plug the modules in to the USB ports as listed below, from left to right.",
+  "module_connect_description": "Power up and plug in the required module(s) via the OT-2 USB Ports. Place the modules as show in the deck map.",
```
Comment (sfoster1) [inline app/src/assets/localization/en/protocol_calibration.json]:
```suggestion
  "module_connect_description": "Power up and plug in the required module(s) via the OT-2 USB Ports. Place the modules as shown in the deck map.",
```

**Review by sfoster1 — CHANGES_REQUESTED** [review]
Little copy thing

**Review by nusrat813 — APPROVED** [review]
Tested - LGTM

**Review by sfoster1 — APPROVED** [review]
Looks good!

**Pull request merged**